src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -37,9 +37,6 @@
   @JsonProperty("Ports") private Map<String, List<PortBinding>> ports;
   @JsonProperty("MacAddress") private String macAddress;
 
-  private NetworkSettings() {
```

Comment:
If you remove this ctor, you should add `@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)` above the class. I think this is why the CircleCI tests are failing with 

```
Caused by: com.fasterxml.jackson.databind.JsonMappingException: No suitable constructor found for type [simple type, class com.spotify.docker.client.messages.NetworkSettings]: can not instantiate from JSON object (missing default constructor or creator, or perhaps need to add/enable type information?)
 at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@40c55905; line: 1, column: 459] (through reference chain: com.spotify.docker.client.messages.ContainerInfo["NetworkSettings"])
```